gensim/utils.py

```diff
@@ -1154,7 +1154,7 @@ def check_output(*popenargs, **kwargs):
     Added extra KeyboardInterrupt handling
     """
     try:
-        process = subprocess.Popen(stdout=subprocess.PIPE, *popenargs, **kwargs)
```

Comment:
can you confirm that ldamallet wrapper works even without the default stdout specified? what prevents you from keeping it as default?